### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by ovinis**

### CODE REVIEW

File: dodo.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import glob
+import os
+import subprocess
+
+LOCALES = ["en","de", "es", "fr", "ga", "it", "ja", "ko", "pt", "sv", "zh"]
+
+def task_mo():
+    """generate a locales .mo file from .po file.   Use task locales_combine to produce final result"""
```
Comment (jacalata) [inline dodo.py:8]: I think this is the same as the code I moved into the file bin/i18n/pygettext.py?

File: tabcmd/locales/de/LC_MESSAGES/de.po [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Tabcmd output strings
```
Comment (jacalata) [inline tabcmd/locales/de/LC_MESSAGES/de.po:1]: I don't think we need to copy this file across with the english strings - it should fall back on english for strings that aren't found for any given language.

**Comment by jacalata** [comment]
I get this error for any language (line position varies):

```
**c:\dev\tabcmd2**>_python -m tabcmd listsites --language fr
Language options:  [None, None, 'en']
Language options:  ['fr', None, 'en']
Failed to load language ' fr ': 'ascii' codec can't decode byte 0xc3 in position 60: ordinal not in range(128)_
```

**Comment by mwhagedorn** [comment]
> I get this error for any language (line position varies):
> 
> ```
> **c:\dev\tabcmd2**>_python -m tabcmd listsites --language fr
> Language options:  [None, None, 'en']
> Language options:  ['fr', None, 'en']
> Failed to load language ' fr ': 'ascii' codec can't decode byte 0xc3 in position 60: ordinal not in range(128)_
> ```

this seems to be fixed now that the header creation issue is sorted

**Pull request merged**